core/src/test/java/com/radixdlt/api/DeterministicCoreApiTestBase.java

```diff
@@ -151,11 +159,51 @@ protected DeterministicTest buildRunningServerTest(int roundsPerEpoch) {
         ProtocolConfig.testingDefault());
   }
 
+  protected DeterministicTest buildRunningServerTest(
+      int roundsPerEpoch,
```

Comment:
We don't need to do this now, but at some point, this is screaming out for a builder pattern :D

Reply:
yeah, I agree, and I am adding this to my list of "refactoring ideas for when I am fed up with foreground tasks" 👍